spec/br_danfe/helper_spec.rb

```diff
@@ -213,4 +213,22 @@
       end
     end
   end
+
+  describe '.format_cep' do
+    it 'returns a formatted CEP' do
+      expect(described_class.format_cep('12345678')).to eq '12.345-678'
```

Comment:
```suggestion
      expect(described_class.format_cep('12345678')).to eql '12.345-678'
```

Já recomendo trocar todos os `eq` nesses testes, se houverem....